src/cli/keepassxc-cli.cpp

```diff
@@ -179,7 +172,7 @@ void enterInteractiveMode(const QStringList& arguments)
 int main(int argc, char** argv)
 {
     if (!Crypto::init()) {
-        qFatal("Fatal error while testing the cryptographic functions:\n%s", qPrintable(Crypto::errorString()));
+        qWarning("Fatal error while testing the cryptographic functions:\n%s", qPrintable(Crypto::errorString()));
```

Comment:
@droidmonkey why warning here? That seems to be at least an error to me.

Reply:
We return fatal, qfatal causes an assert which blocks the return in debug builds.